src/Symfony/Component/Security/Core/Resources/translations/security.de.xlf

```diff
@@ -70,6 +70,14 @@
                 <source>Invalid or expired login link.</source>
                 <target>Ungültiger oder abgelaufener Anmelde-Link.</target>
             </trans-unit>
+            <trans-unit id="19">
+                <source>Too many failed login attempts, please try again in %minutes% minute.</source>
+                <target>Zu viele fehlgeschlagene Anmeldeversuche, bitte versuchen Sie es in einer Minute noch einmal.</target>
```

Comment:
Why do we have a `%minutes%` placeholder in the message but do not reuse it in the translation?

Reply:
Doesn't it sound nicer like that in German for a singular sentence

Reply:
Yes, that's what I thought as well. The value should be `1` at this point.

Reply:
Fine for me if we can be sure that the values is always 1.